.gitlab-ci.yml

```diff
@@ -192,8 +192,8 @@ test_cpp_mac_10_13:
     - build_wheel_mac_10_14_python27
   script:
     - export VIRTUALENV="/Library/Frameworks/Python.framework/Versions/2.7/bin/virtualenv"
-    - (test -d debug && test -d release) || ./configure
-    - cd release/test
+    - (test -d debug && test -d release) || ./configure --no-python
+    - cd debug/test
```

Comment:
Why do we want to use debug rather release?

Reply:
Because debug actually runs all the checks; release does not.  When the NDEBUG flag is passed in, there are numerous internal consistency checks that are turned off, and many of the unit tests in some of the deeper parts of the engine rely on hitting these to get full test coverage.

Reply:
What is the effect on the runtime of the unit tests?

Reply:
It's probably faster to compile + run in debug mode vs. release mode.  They still run in < 15-20 min or so on my desktop -- the CPP tests are much cheaper than the python tests.   I haven't timed the tests in release mode, but the extra compile time required for building in release (2x or so) is comparable to that.